src/BlockTypes/ProductTemplate.php

```diff
@@ -61,7 +61,11 @@ protected function render( $attributes, $content, $block ) {
 		$classnames = '';
 		if ( isset( $block->context['displayLayout'] ) && isset( $block->context['query'] ) ) {
 			if ( isset( $block->context['displayLayout']['type'] ) && 'flex' === $block->context['displayLayout']['type'] ) {
-				$classnames = "is-flex-container columns-{$block->context['displayLayout']['columns']}";
+				if ( $block->context['displayLayout']['shrinkColumns'] ) {
```

Comment:
Hi @kmanijak - I didn't catch this earlier but this can potentially be undefined and will cause a PHP warning. So perhaps use `! empty()`check?